projects/common/src/utilities/validators/email-validator.ts

```diff
@@ -0,0 +1,10 @@
+export const isEmailAddressValid = (emailAddress: string) => {
+  // Regex as per W3C spec for email field
+  // Source: https://html.spec.whatwg.org/multipage/input.html#valid-e-mail-address
```

Comment:
That's a hefty regex. Email validation is always tough since it's been described across various RFCs and there's inconsistencies across some providers, but let's go with it for now. In prior lives, I remember we've stripped down email validation more and more with weird cases in the wild.

Reply:
Yea. Even this one does not cover all the cases. But the w3c spec seemed like a good enough sample.

Reply:
Is it feasible to use the built in angular one? https://angular.io/api/forms/Validators#email looks like it's based on the same thing

Reply:
It is not